src/app.py

```diff
@@ -1464,17 +1524,11 @@ def update_entity(id):
     # normalized_complete_dict = schema_manager.normalize_entity_result_for_response(complete_dict)
 
     # Also reindex the updated entity node in elasticsearch via search-api
-    if entity_dict['entity_type'] in ['Collection']:
-        logger.log(logging.DEBUG
-                   ,f"Skipping re-indexing {entity_dict['entity_type']}"
-                    f" with UUID {entity_dict['uuid']}"
-                    f" until supported by search-api.")
-    else:
-        logger.log(logging.INFO
-                   ,f"Re-indexing for creation of {entity_dict['entity_type']}"
-                    f" with UUID {entity_dict['uuid']}")
+    logger.log(logging.INFO
+               ,f"Re-indexing for creation of {entity_dict['entity_type']}"
```

Comment:
@kburke the logging message needs a bit tweak here since the entity gets updated in this `update_entity()`